Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.12-slim
+
+LABEL maintainer="vladislav.tmf@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /src
+
+COPY requirements.txt requirements.txt
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```

Comment:
Verify that the file path `app/main.py` exists in your project structure and is the correct entry point for your application.